### PR TITLE
LIP0053: fix typo and improve default events topic comments

### PR DESCRIPTION
**Pull request by ricott1**
Closes https://github.com/LiskHQ/lips/issues/324. Closes https://github.com/LiskHQ/lips/issues/309.

### CODE REVIEW

File: proposals/lip-0053.md [hunk 1]
```diff
@@ -522,7 +522,8 @@ def execute(trs: Transaction) -> None:
     for ccmBytes in crossChainMessages:
         ccmID = sha256(ccmBytes)
         # Set ccmID (instead of the transaction ID) as default topic of this event.
-        defaultEventsTopic = ccmID
+        # The default topic is added automatically by emitEvent.
+        eventsDefaultTopic = ccmID
```
Comment (AndreasKendziorra) [inline proposals/lip-0053.md:526]: Now I'm just wondering what's the scope of this change? I mean, when will the transaction ID be the default topic again? After the next event? After the for loop? After the function? Never? Maybe this should be clarified in the comment as well. Or we add an explicit `eventsDefaultTopic = transactionID` in the right place.
Depending on the answer, we may also be able to drop the 2nd `eventsDefaultTopic = ccmID` in this function.

File: proposals/lip-0053.md [hunk 2]
```diff
@@ -732,7 +735,8 @@ def execute(trs: Transaction) -> None:
     for ccmBytes in crossChainMessages:
         ccmID = sha256(ccmBytes)
         # Set ccmID (instead of the transaction ID) as default topic to all events emitted in apply.
-        defaultEventsTopic = ccmID
+        # The default topic is added automatically by emitEvent.
+        eventsDefaultTopic = ccmID
```
Comment (AndreasKendziorra) [inline proposals/lip-0053.md:739]: Same as above: could potentially removed.

**Review by AndreasKendziorra — CHANGES_REQUESTED** [review]

File: proposals/lip-0053.md (hunk 2 above)
Comment (ricott1) [reply proposals/lip-0053.md:743]: Notice that ccmID changes during the loop, so we can't remove this

File: proposals/lip-0053.md (hunk 1 above)
Comment (ricott1) [reply proposals/lip-0053.md:526]: I explicitly added the transaction ID back as default event

File: proposals/lip-0053.md [hunk 3]
```diff
@@ -593,6 +593,10 @@ def execute(trs: Transaction) -> None:
     # Update the context to indicate that now we stop the CCM processing.
     ctx.ccmProcessing = False
 
+    # Set the transaction ID back as default topic.
+    # The default topic is added automatically by emitEvent.
+    eventsDefaultTopic = transaction ID
```
Comment (AndreasKendziorra) [inline proposals/lip-0053.md:598]: 1. I would do this for LIP 0054 as well.
2. Maybe it's a bit cleaner to have than only a single `eventsDefaultTopic = ccmID` statement before the first for loop. But it's also ok to have like this if you prefer.

**Review by AndreasKendziorra — CHANGES_REQUESTED** [review]

File: proposals/lip-0053.md (hunk 3 above)
Comment (AndreasKendziorra) [reply proposals/lip-0053.md:598]: ah, forget the 2nd point. I just read your other comment.

File: proposals/lip-0053.md (hunk 2 above)
Comment (AndreasKendziorra) [reply proposals/lip-0053.md:743]: right, I missed this

File: proposals/lip-0053.md (hunk 3 above)
Comment (ricott1) [reply proposals/lip-0053.md:598]: In LIP54 there are no other events emitted in the transaction execution, that's why I didn't add it

Comment (AndreasKendziorra) [reply proposals/lip-0053.md:598]: ok, I see.

**Review by AndreasKendziorra — APPROVED** [review]

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**